packages/client/hmi-client/src/petrinet/petrinet-renderer.ts

```diff
@@ -12,6 +12,8 @@ const pathFn = d3
 	.y((d) => d.y);
 
 export class PetrinetRenderer extends graphScaffolder.BasicRenderer<NodeData, EdgeData> {
+	displayedHandles: D3SelectionHandles<NodeData> | null = null;
```

Comment:
It might be cleaner to cache the node instead of the node's handles, just so there aren't too many points of references that point to the same DOM/data-binds.

```
nodeSelection: D3SelectionINode<NodeData> | null = null
...
on('node-click',  (_eventName, _event, selection: D3SelectionINode<NodeData>)  => {
  if (nodeSelection) nodeSelection.selectAll('.blahblah').transition().duration(200) ....  
  nodeSelection = selection;
  ndoeSelection.selectAll('.blahblah').transition().duration(200) ...
})

Reply:
ok yes thats a good idea